hr_attendance_geolocation/static/src/js/geolocation_user_context_legacy.esm.js

```diff
@@ -0,0 +1,34 @@
+/** @odoo-module **/
+/* Copyright 2023 Alexandre D. Díaz - Grupo Isonor
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+import {registry} from "@web/core/registry";
+import * as legacySession from "web.session";
+
+const LegacyGelocationSessionUserContextService = {
+    dependencies: ["user"],
+
+    start(env) {
+        Object.defineProperty(legacySession, "user_geolocation_context", {
+            set: (values) => {
+                const nvals = {};
+                if ("longitude" in values) {
+                    nvals.longitude = values.longitude;
+                }
+                if ("latitude" in values) {
+                    nvals.latitude = values.latitude;
+                }
+                if (!_.isEmpty(nvals)) {
+                    env.services.user.updateContext(nvals);
+                }
```

Comment:
We can simplify it using destructuring:
```suggestion
                const {latitude, longitude} = vals;
                if (latitude && longitude) {
                    env.services.user.updateContext({
                        latitude: latitude,
                        longitude: longitude,
                    });
                }
```

Reply:
We don't currently use it... but if we make this change, the values cannot be updated individually.

Reply:
Why would you?